### ISSUE TITLE
Forwarded messages don't have inline images shown

### ISSUE THREAD

**Issue by chenhuigang** [thunderbird/thunderbird-android#2490]
Please search to check for an existing issue (including closed issues, for which the fix may not have yet been released) before opening a new issue: https://github.com/k9mail/k-9/issues?q=is%3Aissue

### Expected behavior
 Some images can show in the text part

### Actual behavior
Some images can't show in the text part

### Steps to reproduce
1. Create a message on PC and add some images in the text part , forward the message  and receive it
2. when I read the text part in  message detail 
3. Those images is not show in the text part

### Environment
K-9 Mail version: 5.206

Android version:Android 6.0

Account type (IMAP, POP3, WebDAV/Exchange):IMAP

**Comment by philipwhiuk** [comment]
Can you provide source of an affected email. Also can you confirm you clicked 'Download complete message' if it was shown?

**Comment by chenhuigang** [comment]
I clicked on these two button and checked the message just now,that still problem.Thank you for your answer.

**Comment by chenhuigang** [comment]
Thank you for your answer. Check the message after  'Download complete message' 'Show pictures'  has checked ,that still problem.

**Comment by philipwhiuk** [comment]
Please provide the emails source then so we can reproduce the issue

**Comment by chenhuigang** [comment]
I don't quite understand the meaning of the emails source，provide my email account password 
 or some inline images of the problem

**Comment by daquexian** [comment]
Glad to see a Chinese user of K-9 :D. You can forward the message to me if you want, so that I can help reproduce it
My email: zhjh123@mail.ustc.edu.cn

**Comment by chenhuigang** [comment]
I have forward the message to zhjh123@mail.ustc.edu.cn  .   Please check for proper receipt.

**Comment by xenithorb** [comment]
I can confirm this problem. What's worse is that certain inline images don''t show up as attachments, and therefore they can't be viewed at all. ALL inline images (both inline attachments and the regular non-attached inline variety) forward in a broken state when using K9 mail. In the case of the image being attached AND inline, the attached is forwarded though and the user can display the image separately, it's just not displayed in the image like it's supposed to. 

In short, K9 is mangling inline images when forwarding and it's breaking things.

**Comment by iwsbarker** [comment]
I can also confirm this. I received a message with images in the message, and the images were attachments to the email. When I forwarded the email to someone else, the attached images were not forwarded with the message, even though they were shown in the forwarded message as I composed it.

**Comment by xenithorb** [comment]
Ping?

**Comment by anthony-sticky** [comment]
I reproduce this problem with the current K9 version

**Comment by cketti** [comment]
Can someone please attach the source of a message that won't be displayed properly when it is forwarded?
